src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAccountTransactions.java

```diff
@@ -0,0 +1,156 @@
+	 */
+	public LookupAccountTransactions afterTime(java.util.Date afterTime) {
+		addQuery("after-time", new java.text.SimpleDateFormat("yyyy-MM-dd'T'h:m:ssZ").format(afterTime));
+		return this;
+	}
+
+	/**
+	 * Asset ID 
+	 */
+	public LookupAccountTransactions assetId(Long assetId) {
+		addQuery("asset-id", String.valueOf(assetId));
+		return this;
+	}
+
+	/**
+	 * Include results before the given time. Must be an RFC 3339 formatted string. 
+	 */
+	public LookupAccountTransactions beforeTime(java.util.Date beforeTime) {
+		addQuery("before-time", new java.text.SimpleDateFormat("yyyy-MM-dd'T'h:m:ssZ").format(beforeTime));
+		return this;
+	}
+
+	/**
+	 * Results should have an amount greater than this value. MicroAlgos are the 
+	 * default currency unless an asset-id is provided, in which case the asset will be 
+	 * used. 
+	 */
+	public LookupAccountTransactions currencyGreaterThan(Long currencyGreaterThan) {
+		addQuery("currency-greater-than", String.valueOf(currencyGreaterThan));
+		return this;
+	}
+
+	/**
+	 * Results should have an amount less than this value. MicroAlgos are the default 
+	 * currency unless an asset-id is provided, in which case the asset will be used. 
+	 */
+	public LookupAccountTransactions currencyLessThan(Long currencyLessThan) {
+		addQuery("currency-less-than", String.valueOf(currencyLessThan));
+		return this;
+	}
+
+	/**
+	 * Maximum number of results to return. 
+	 */
+	public LookupAccountTransactions limit(Long limit) {
+		addQuery("limit", String.valueOf(limit));
+		return this;
+	}
+
+	/**
+	 * Include results at or before the specified max-round. 
+	 */
+	public LookupAccountTransactions maxRound(Long maxRound) {
+		addQuery("max-round", String.valueOf(maxRound));
+		return this;
+	}
+
+	/**
+	 * Include results at or after the specified min-round. 
+	 */
+	public LookupAccountTransactions minRound(Long minRound) {
+		addQuery("min-round", String.valueOf(minRound));
+		return this;
+	}
+
+	/**
+	 * The next page of results. Use the next token provided by the previous results. 
+	 */
+	public LookupAccountTransactions next(String next) {
+		addQuery("next", String.valueOf(next));
+		return this;
+	}
+
+	/**
+	 * Specifies a prefix which must be contained in the note field. 
+	 */
+	public LookupAccountTransactions notePrefix(String notePrefix) {
+		addQuery("note-prefix", String.valueOf(notePrefix));
+		return this;
+	}
+
+	/**
+	 * Include results for the specified round. 
+	 */
+	public LookupAccountTransactions round(Long round) {
+		addQuery("round", String.valueOf(round));
+		return this;
+	}
+
+	/**
+	 * SigType filters just results using the specified type of signature: 
+	 *   sig - Standard 
+	 *   msig - MultiSig 
+	 *   lsig - LogicSig 
+	 */
+	public LookupAccountTransactions sigType(String sigType) {
+		addQuery("sig-type", String.valueOf(sigType));
+		return this;
+	}
+
+	/**
+	 * Lookup the specific transaction by ID. 
+	 */
+	public LookupAccountTransactions txId(Address txId) {
```

Comment:
I think I was wrong about this one, it should be a string after all.

Reply:
reverted to string.